home/management/commands/import_csv_content.py

```diff
@@ -86,16 +156,14 @@ def clean_row(row):
             reader = csv.DictReader(f)
             for row in reader:
                 row = clean_row(row)
-                contentpage = ContentPage(
-                    title=row["web_title"],
-                    subtitle=row["web_subtitle"],
-                    body=get_rich_text_body(row["web_body"]),
-                    enable_whatsapp=True,
-                    whatsapp_title=row["whatsapp_title"],
-                    whatsapp_body=get_text_body(row["whatsapp_body"]),
-                )
-                create_tags(row, contentpage)
-                add_parent(row, contentpage, home_page)
-                contentpage.save_revision().publish()
+                contentpage = add_web(row)
+                contentpage = add_whatsapp(row, contentpage)
+                contentpage = add_messenger(row, contentpage)
+                contentpage = add_viber(row, contentpage)
+
+                if contentpage:
+                    create_tags(row, contentpage)
+                    add_parent(row, contentpage, home_page)
+                    contentpage.save_revision().publish()
```

Comment:
You might want to log something if a content page hasn't been created at this point